yb-voyager/src/query/sqltransformer/transformer.go

```diff
@@ -146,6 +147,7 @@ func (t *Transformer) MergeConstraints(stmts []*pg_query.RawStmt) ([]*pg_query.R
 				if !ok {
 					return nil, fmt.Errorf("CREATE TABLE stmt not found for table %v", objectName)
 				}
+				log.Infof("merging constraint %v into CREATE TABLE for object %v", constrType, objectName)
```

Comment:
can this be debug log?

Reply:
this might help in debugging so kept it for default log level.